plugins/woocommerce/client/admin/client/payments/test/payment-recommendations.test.tsx

```diff
@@ -56,7 +57,27 @@ jest.mock( '../../lib/notices', () => ( {
 	} ),
 } ) );
 
+declare global {
+	interface Window {
+		wcAdminFeatures: Record< string, boolean >;
+	}
+}
+
 describe( 'Payment recommendations', () => {
+	afterEach( () => {
+		window.wcAdminFeatures[ 'reactify-classic-payments-settings' ] = false;
```

Comment:
Do we have a constant for this feature's name? It appears to be used in multiple places throughout the codebase.

Reply:
We don't. In our codebase I noticed this approach and wanted to be consistent
<img width="467" alt="image" src="https://github.com/user-attachments/assets/58b0844d-1b8a-4e1f-8591-638828955312">

Reply:
Perfect, thanks for looking. In that case, please ignore my nitpick :)